software/docker/kafka-can-log/install-sh

```diff
@@ -0,0 +1,508 @@
+#!/bin/sh
```

Comment:
I don't think this is supposed to be committed. We should add/update `.gitignore`